src/lib.rs

```diff
@@ -29,6 +33,34 @@ pub fn generate_keypair_and_auth_secret(
     Ok((local_key_pair, auth_secret))
 }
 
+/// Encrypt a block using default AES128GCM encoding.
+///
+/// param remote_pub &[u8] - The remote public key
+/// param remote_auth &u8 - The remote authorization token
+/// param salt &[u8] - The locally generated random salt
+/// param data &[u8] - The data to encrypt
+///
+pub fn encrypt(remote_pub: &[u8], remote_auth: &[u8], salt: &[u8], data: &[u8]) -> Result<Vec<u8>> {
+    let remote_key = crypto_backends::CryptoImpl::public_key_from_raw(remote_pub)?;
+    let local_key = LocalKeyPairImpl::generate_random()?;
+    // TODO: randomize pad.
```

Comment:
Is this `TODO` out-of-date? It appears to be random.

Reply:
It is. Thanks!